Node/calling/src/bots/CallConnector.ts

```diff
@@ -198,7 +199,7 @@ export class CallConnector implements ucb.ICallConnector, bs.IBotStorage {
 
                     // verify appId, issuer, token expirs and token notBefore
                     if (decoded.payload.aud != this.settings.appId || decoded.payload.iss != issuer || 
-                        now > decoded.payload.exp || now < decoded.payload.nbf) {
+                        (now - CLOCK_TOLERANCE) > decoded.payload.exp || (now + CLOCK_TOLERANCE) < decoded.payload.nbf) {
```

Comment:
you can remove  `(now - CLOCK_TOLERANCE) > decoded.payload.exp || (now + CLOCK_TOLERANCE) < decoded.payload.nbf)` check from here and rely on the clockTolerance check done by jsonwebtoken module.